### ISSUE TITLE
Generate a html block for my page / tree starting page with all defined settings

### ISSUE THREAD

**Issue by hartenthaler** [Neriderc/GVExport#450]
It would be nice to have such a dynamically generated block available which can be added to the "my page" or the starting page of a tree. See as an example: https://github.com/JustCarmen/webtrees-fancy-treeview/issues/121

**Comment by Neriderc** [comment]
Can you explain how to trigger the option in Fancy treeview?

I haven't used this module until now, and I'm not sure where to find the option you're referring to.

**Comment by hartenthaler** [comment]
After installing the newest version 2.0.2 of that module there is a new type of home page block available:
![Screenshot (406)](https://github.com/Neriderc/GVExport/assets/6324175/89be907d-2006-4b87-90a4-e79876cb5ea9)

Then you can add that block to the home page of a tree and see the links:
![Screenshot (405)](https://github.com/Neriderc/GVExport/assets/6324175/8d9bbf95-ddcd-4ced-8e62-d2a22cf59ed3)

You can configure the module in the Control panel / Modules / Genealogy / Menues, but this is not necessary. Go to the page of an individual and select the "edit" button in the top/right corner. Select "Create link to ancestors/descendants page". Then you should see the link list in the new home page block of your tree.

**Comment by Neriderc** [comment]
Ah thanks, I see now. I will have to have a think about how we might do this. We can make a list, but a list is not very useful without a link. Do we only want to list the settings where a sharing URL has been created?

And this would be just for a user's home page. For the tree home page, I am not sure what we could automatically generate here. Different people can see different things. And if settings have been shared with a URL, we do not know who the user intended to share them with. I would think that we should not be automatically generating a list of shared settings to show on a tree home page, as I don't know how we would determine what to put on it. Thoughts?

**Comment by hartenthaler** [comment]
I agree, a list without links is not useful. A link list on the user's home page would be nice. 

For the tree's home page: this is much more complicated. The list depends if a user is logged in or if he/she is a visitor. And you are right, that you cannot know for whom a link was generated. This would require a new function when generating a settings link: "generate a public URL to this setting for this tree". And then show all the generated public URL links in the new tree home page block. Maybe, this is too complicated.

**Comment by Neriderc** [comment]
Just having a think about this.

Let's say we add an option where on a saved setting, you can click "Add saved setting to tree home page" or "Add saved setting to My page".

Is this any different from the current options, where a user can add to the favourites in My page, and an admin can also add to the Favourites in the tree home page?
